ui/src/app/pages/admin/admin-workspace.tsx

```diff
@@ -237,88 +283,29 @@ class AdminWorkspaceImpl extends React.Component<UrlParamsProps, State> {
           </div>
           <h3>Cohort Builder</h3>
           <div className='cohort-builder' style={{marginTop: '1rem'}}>
-            {
-              this.workspaceInfoField(
-                '# of Cohorts',
-                resources.workspaceObjects.cohortCount
-              )
-            }
-            {
-              this.workspaceInfoField(
-                '# of Concept Sets',
-                resources.workspaceObjects.conceptSetCount
-              )
-            }
-            {
-              this.workspaceInfoField(
-                '# of Data Sets',
-                resources.workspaceObjects.datasetCount
-              )
-            }
+            {workspaceInfoField('# of Cohorts', resources.workspaceObjects.cohortCount)}
+            {workspaceInfoField('# of Concept Sets', resources.workspaceObjects.conceptSetCount)}
+            {workspaceInfoField('# of Data Sets', resources.workspaceObjects.datasetCount)}
           </div>
           <h3>Cloud Storage Objects</h3>
           <div className='cloud-storage-objects' style={{marginTop: '1rem'}}>
-            {
-              this.workspaceInfoField(
-                'GCS bucket path',
-                resources.cloudStorage.storageBucketPath
-              )
-            }
-            {
-              this.workspaceInfoField(
-                '# of Notebook Files',
-                resources.cloudStorage.notebookFileCount
-              )
-            }
-            {
-              this.workspaceInfoField(
-                '# of Non-Notebook Files',
-                resources.cloudStorage.nonNotebookFileCount
-              )
-            }
-            {
-              this.workspaceInfoField(
-                'Storage used (bytes)',
-                resources.cloudStorage.storageBytesUsed
-              )
-            }
+            {workspaceInfoField('GCS bucket path', resources.cloudStorage.storageBucketPath)}
+            {workspaceInfoField('# of Workbench-managed notebook files', resources.cloudStorage.notebookFileCount)}
```

Comment:
changed this text to be more precise.  See the comments at https://precisionmedicineinitiative.atlassian.net/browse/RW-5471

Reply:
FYI: Technically there is a middle-ground, which are notebooks in subdirs of notebooks/. These are synchronized back/forth to the notebook VM/GCS, but are not displayed on the RW notebooks list page. Way too much detail for this line, and I don't think I would change the description you have here.